### ISSUE TITLE
low contrast on logout button mouseover

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#249]
It does not have a good contrast ratio. Invert colors.

**Comment by giovanisp** [comment]
hi @derekpa can you compile LESS and put on test to check? thanks.

**Comment by derekpa** [comment]
Done. Looks nicer.
